ush/config.sh.aws_cloud

```diff
@@ -1,79 +1,84 @@
-ACCOUNT=zrtrr
-
-MACHINE=LINUX
-WORKFLOW_MANAGER=rocoto
-SCHED=slurm
-
-LMOD_PATH=/apps/lmod/lmod/init/sh
-BUILD_ENV_FN=build_hera_intel.env
-
-RUN_CMD_UTILS="srun"
-RUN_CMD_FCST="srun"
-RUN_CMD_POST="srun"
-
-NCORES_PER_NODE=40
-PARTITION_DEFAULT="hera"
-QUEUE_DEFAULT=batch
-PARTITION_HPSS=service
-QUEUE_HPSS=batch
-PARTITION_FCST=hera
-QUEUE_FCST=batch
-
-EXPT_BASEDIR=/scratch2/BMC/wrfruc/cholt
-EXPT_SUBDIR=rrfs_cloud
-
-RUN=RRFS_CONUS
-
-# Copied directly from setup.sh HERA settings
-FIXgsm=${FIXgsm:-"/scratch1/NCEPDEV/global/glopara/fix/fix_am"}
-TOPO_DIR=${TOPO_DIR:-"/scratch1/NCEPDEV/global/glopara/fix/fix_orog"}
-SFC_CLIMO_INPUT_DIR=${SFC_CLIMO_INPUT_DIR:-"/scratch1/NCEPDEV/global/glopara/fix/fix_sfc_climo"}
-FIXLAM_NCO_BASEDIR=${FIXLAM_NCO_BASEDIR:-"/scratch2/BMC/det/FV3LAM_pregen"}
-
-
-# Get external model files
-USE_USER_STAGED_EXTRN_FILES=TRUE
-EXTRN_MDL_SOURCE_BASEDIR_ICS="/scratch2/BMC/wrfruc/cholt/data/gfs"
-EXTRN_MDL_FILES_ICS=( "gfs.t00z.pgrb2.0p25.f000")
-EXTRN_MDL_SOURCE_BASEDIR_LBCS="/scratch2/BMC/wrfruc/cholt/data/gfs"
-EXTRN_MDL_FILES_LBCS=( \
-  "gfs.t00z.pgrb2.0p25.f006"
-  "gfs.t00z.pgrb2.0p25.f012"
-)
-FV3GFS_FILE_FMT_ICS=grib2
-FV3GFS_FILE_FMT_LBCS=grib2
+MACHINE="LINUX"
+ACCOUNT="an_account"
+WORKFLOW_MANAGER="rocoto"
+SCHED="slurm"
+
+EXPT_BASEDIR="/lustre"
+EXPT_SUBDIR="test_rrfs"
+
+RUN_CMD_UTILS="srun --mpi=pmi2"
+RUN_CMD_FCST="srun --mpi=pmi2"
```

Comment:
I was thinking this setting landed on mpirun instead.

Reply:
There was some initial confusion about this.  But, I think the problem was that I wasn't using the correct Intel MPI module.   And that has been corrected in NOAA-GSL/ufs-srweather-app#16.  Now, I am getting much better timings and no hangs.